plenum/server/replica.py

```diff
@@ -2231,23 +2250,21 @@ def _gc(self, till3PCKey):
         self.logger.info("{} cleaning up till {}".format(self, till3PCKey))
         tpcKeys = set()
         reqKeys = set()
-        for key3PC, pp in self.sentPrePrepares.items():
-            if compare_3PC_keys(till3PCKey, key3PC) <= 0:
-                tpcKeys.add(key3PC)
-                for reqKey in pp.reqIdr:
-                    reqKeys.add(reqKey)
-        for key3PC, pp in self.prePrepares.items():
+
+        for key3PC, pps in self.pre_prepare_tss.items():
```

Comment:
Do we store PPs created by a primary in `pre_prepare_tss`? I don't think so. It means that all send PPs will not be GCed.